docs/documentation/upgrading/topics/changes/changes-26_5_0.adoc

```diff
@@ -141,6 +141,15 @@ See the link:{upgradingguide_link}[{upgradingguide_name}] for details on how to
 
 When enabling authorization on a new client (creating a Resource Server), Keycloak no longer automatically creates a "Default Resource", a "Default Policy," and a "Default Permission."
 
+=== New logic to delete expired sessions from the database
+
+New logic has been implemented to delete expired sessions from the database.
+Previously, a cluster-aware scheduled task triggered the removal, where only a single Keycloak instance would check and remove expired entries.
+
+In the new version, each Keycloak instance handles different realms.
```

Comment:
Maybe we can make this a bit more detailed, e.g.:

"In the new version removal is handled by all Keycloak instances, with each instance handling the expiration of sessions in different realms".